cif/products/src/helpers/CommerceIntegrationHelper.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 diconium
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default class CommerceIntegrationHelper {
+
+  static checkRequestType(filter: any) {
+    if (filter.startsWith('variants.sku')) {
+      return CIFRequestType.Variant;
+    }
+
+    if (filter.startsWith('categories.id')) {
+      return CIFRequestType.Category;
+    }
+
+  }
+
+  static isCIFRequest = (mappable: any): boolean => (mappable.filter) ? true : false;
```

Comment:
why do you pass the complete mappable object if you only need the filter property?

Reply:
If you see the below comment probably this method can be deleted

Reply:
removed.